flight/Libraries/math/misc_math.h

```diff
@@ -78,6 +78,100 @@ static inline bool IS_NOT_FINITE(float x) {
 	return (!isfinite(x));
 }
 
+/* Following functions from fastapprox https://code.google.com/p/fastapprox/
```

Comment:
No, it shouldn't be a separate file. There are hundreds of files already in the code that include a 3-clause BSD license, this one shouldn't be treated any differently. Just put the boilerplate in the file and the general disclaimer in LICENSE.txt covers it.